### ISSUE TITLE
Critical Issue (Loss Function Error)

### ISSUE THREAD

**Issue by Joo-Hyun-Lee** [yunjey/mnist-svhn-transfer#4]
solver.py 128, 134 .. line 

- Before (Now)
d1_loss = torch.mean((out-1)**2) 

- After
d1_loss = torch.mean((out-1).clamp(max=0)**2)
